Sources/DocCHTML/MarkdownRenderer+Parameters.swift

```diff
@@ -0,0 +1,143 @@
+        var items: [XMLElement] = []
+        items.reserveCapacity(parameterInfo.count * 2)
+        for parameter in parameterInfo {
+            // name
+            items.append(
+                .element(named: "dt", children: [
+                    .element(named: "code", children: [.text(parameter.name)])
+                ])
+            )
+            // description
+            items.append(
+                .element(named: "dd", children: parameter.content.map { visit($0) })
+            )
+        }
+        
+        return items
+    }
+    
+    private func _dualLanguageParameters(
+        primary:   (key: SourceLanguage, value: [ParameterInfo]),
+        secondary: (key: SourceLanguage, value: [ParameterInfo])
+    ) -> XMLElement {
+        // "Shadow" the parameters with more descriptive tuple labels
+        let primary   = (language: primary.key,   parameters: primary.value)
+        let secondary = (language: secondary.key, parameters: secondary.value)
+        
+        // When there are exactly two language representations, which is very common,
+        // avoid duplication and only create `<dt>` and `<dd>` HTML elements _once_ if the parameter exist in both language representations.
+        
+        // Start by rendering the primary language's parameter, then update that list with information about language-specific parameters.
+        var items = _singleLanguageParameterItems(primary.parameters)
+        
+        // Find all the inserted and deleted parameters.
+        // This assumes that parameters appear in the same _order_ in each language representation, which is true in practice.
+        // If that assumption is wrong, it will produce correct looking results but some repeated markup.
+        let differences = secondary.parameters.difference(from: primary.parameters, by: { $0.name == $1.name })
+        
+        // Track which parameters _only_ exist in the primary language in order to insert the secondary languages's _unique_ parameters in the right locations.
+        var primaryOnlyIndices = Set<Int>()
+        
+        // Add a "class" attribute to the parameters that only exist in the secondary language representation.
+        // Through CSS, the rendered page can show and hide HTML elements that only apply to a specific language representation.
+        for case let .remove(offset, _, _) in differences.removals {
+            // This item only exists in the primary parameters
+            primaryOnlyIndices.insert(offset)
+            let index = offset * 2
+            // Mark those items as only being applying to the first language
+            items[index    ].addAttributes(["class": "\(primary.language.id)-only"])
+            items[index + 1].addAttributes(["class": "\(primary.language.id)-only"])
+        }
+        
+        // Insert parameter that only exist in the secondary language representation.
```

Comment:
```suggestion
        // Insert parameter that only exists in the secondary language representation.
```

Reply:
Thanks. This is one of those things that I can never get right 🙃